tide_core.install

```diff
@@ -587,3 +587,49 @@ function tide_core_update_8021() {
   $config_storage = \Drupal::service('config.storage');
   $config_storage->write($view, $source->read($view));
 }
+
+/**
+ * Enable scheduled transitions for each content type.
+ */
+function tide_core_update_8022() {
+  // Get the available content types.
+  $types = \Drupal::entityTypeManager()
+  ->getStorage('node_type')
+  ->loadMultiple();
+  foreach ($types as $type) {
+    if ($type->id()) {
+      $content_types[] = $type->id();
+    }
+  }
+  if (\Drupal::moduleHandler()->moduleExists('scheduled_transitions') && !empty($content_types)) {
+    $config_factory = \Drupal::configFactory();
+    $config = $config_factory->getEditable('scheduled_transitions.settings');
+    $bundles = $config->get('bundles');
+    if (!$bundles) {
+      foreach ($content_types as $bundle) {
+        $bundles_array[] = ['entity_type' => 'node', 'bundle' => $bundle];
+      }
+      $config->set('bundles', $bundles_array);
+      $config->save();
+    }
+  }
+}
+
+/**
+ * Import sdpa password policy.
+ */
+function tide_core_update_8023() {
```

Comment:
This belongs in dpc-sdp/baywatch because it's SDP specific i.e as an open source project implementations of tide_core may have very different requirements to SDP.

Reply:
I am going to move this hook to baywatch module.